cmd/clusterctl/examples/aws/machines.yaml.template

```diff
@@ -4,7 +4,7 @@ items:
   - apiVersion: "cluster.k8s.io/v1alpha1"
     kind: Machine
     metadata:
-      generateName: aws-controlplane-
+      name: aws-controlplane-${CLUSTER_NAME}
```

Comment:
I would suggest using a static index here rather than `${CLUSTER_NAME}`.

We really should be prefixing the aws instance `Name` tags with the cluster name rather than just passing in the machine name.